app/auth/views.py

```diff
@@ -36,6 +36,25 @@ def post():
                         'message': 'Wrong email format. Check your Email.'
                     }
                     return make_response(jsonify(response)), 403
+                # print('your password is {}'.format(password))
+                count = 1
+                prev = ''
+                for letter in password:
```

Comment:
instead of looping over the password, you can use `" " in password` to check if there is white space in the password. it will return `True` if there is white space.

Reply:
I could, although am using it to also check if there is a repetition of many letters in the password